proc/community/src/test/java/org/neo4j/gds/labelpropagation/LabelPropagationStreamProcTest.java

```diff
@@ -78,6 +84,32 @@ void testEstimate() {
         )));
     }
 
+    static Stream<Arguments> communitySizeInputs() {
+        return Stream.of(
+                Arguments.of(Map.of("minCommunitySize", 1), RESULT.toArray(RESULT.toArray(new Long[0]))),
```

Comment:
```suggestion
                Arguments.of(Map.of("minCommunitySize", 1), RESULT.toArray(new Long[0])),
```

Reply:
I felt so embarrassed when I first read this 😳

Reply:
No worries, I also had to look twice if this isnt doing anything unexpected. I mean it did work for the test :sweat_smile: